straxen/url_config.py

```diff
@@ -47,8 +57,13 @@ def __init__(self, cache=0, **kwargs):
             self.final_type = self.type
             self.type = OMITTED # do not enforce type on the URL
         if cache:
-            maxsize = cache if isinstance(cache, int) else None
-            self.dispatch = lru_cache(maxsize)(self.dispatch)
+            cache_len = 1_000_000 if cache is True else int(cache) 
```

Comment:
Sorry, I am a bit confused right now. Is not the default size a bit long? Can you have more than one entry per URL per run? Even the gains should be just one entry (1 x 494) is not it?

Reply:
changed it to 100